dmoj/urls.py

```diff
@@ -28,62 +27,62 @@
 admin.autodiscover()
 
 register_patterns = [
-    url(r'^activate/complete/$',
+    path('activate/complete/',
         TitledTemplateView.as_view(template_name='registration/activation_complete.html',
                                    title='Activation Successful!'),
         name='registration_activation_complete'),
     # Activation keys get matched by \w+ instead of the more specific
     # [a-fA-F0-9]{40} because a bad activation key should still get to the view;
     # that way it can return a sensible "invalid key" message instead of a
     # confusing 404.
-    url(r'^activate/(?P<activation_key>\w+)/$',
+    re_path(r'^activate/(?P<activation_key>\w+)/$',
```

Comment:
Let use `<str:activation_key>`, since we want failed matches to show up as "invalid key" instead of generic 404.